data-products/_example/03-transformations.yml

```diff
@@ -0,0 +1,16 @@
+# Specify and describe any transformations or cleansing the data has undergone.
+# The types of cleansing we recognise are described in our documentation.
+cleansing:
```

Comment:
What if you transpose these? This could be more intuitive and user friendly.

population_by_offence:
   cleansing: "none"
   transformation: "none"

Reply:
Yes! Absolutely better, thanks 👍